styles.css

```diff
@@ -0,0 +1,53 @@
+html{
+    font-size: 12px;
+    background-color: beige;
+}
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+}
+h1{
+    font-size: 4rem;
+    margin bottom: 0;

```

Comment:
it would be better to add a "-" between the margin and the bottom so that your page looks like you intended to.